<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWin</class>
 <widget class="QMainWindow" name="MWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>嗶哩嗶哩盒子©Lewis Tian</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/bililogo</normaloff>:/images/bililogo</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>35</width>
    <height>35</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="home">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,3,8">
        <item>
         <layout class="QHBoxLayout" name="layout2">
          <item>
           <widget class="QLabel" name="logo">
            <property name="minimumSize">
             <size>
              <width>220</width>
              <height>105</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>105</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hpacer3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="vspacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout1" stretch="1,1,4,0,1">
          <item>
           <spacer name="hspace1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxHome">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>视频</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>图片</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHome">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="searchBtn">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>搜索</string>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hspace2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="vspacer1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>218</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="vpage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QToolButton" name="v2homeBtn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>回首页</string>
             </property>
             <property name="text">
              <string>首页</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="dlPathBtn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>下载目录：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="hspacer3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>739</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="vdownloadBtn">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>32</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>下载选中行的视频</string>
             </property>
             <property name="text">
              <string>下载</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="dlAllBox">
             <property name="toolTip">
              <string>下载全部视频</string>
             </property>
             <property name="text">
              <string>下载全部</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="vtable">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>#</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>当前分p标题</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>时长</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>大小</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下载进度</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>切片</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>标题</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ppage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Setting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="minSizeAction"/>
    <addaction name="bgiPathAction"/>
    <addaction name="downloadPathaction"/>
   </widget>
   <widget class="QMenu" name="About">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="authorAction"/>
    <addaction name="appAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="vPageAction"/>
    <addaction name="pPageAction"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="Setting"/>
   <addaction name="About"/>
  </widget>
  <action name="minSizeAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/min.png</normaloff>:/images/min.png</iconset>
   </property>
   <property name="text">
    <string>窗口最小尺寸</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="bgiPathAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/bgm.png</normaloff>:/images/bgm.png</iconset>
   </property>
   <property name="text">
    <string>背景图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="authorAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/me.png</normaloff>:/images/me.png</iconset>
   </property>
   <property name="text">
    <string>开发者</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="appAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/app.png</normaloff>:/images/app.png</iconset>
   </property>
   <property name="text">
    <string>本程序</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="downloadPathaction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/downloadpath.png</normaloff>:/images/downloadpath.png</iconset>
   </property>
   <property name="text">
    <string>下载路径</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="vPageAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/v.png</normaloff>:/images/v.png</iconset>
   </property>
   <property name="text">
    <string>视频下载页</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="pPageAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/bgm.png</normaloff>:/images/bgm.png</iconset>
   </property>
   <property name="text">
    <string>图片下载页</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/car.png</normaloff>:/images/car.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
